doc/command-line-flags.md

```diff
@@ -246,6 +246,9 @@ Allows `gh-ost` to connect to the MySQL servers using encrypted connections, but
 
 `--ssl-key=/path/to/ssl-key.key`: SSL private key file (in PEM format).
 
+### storage-engine
```

Comment:
@wangzihuacool / @dm-2 I think we should explain more about the limitations in testing of `gh-ost` on RocksDB and mark this support as "experimental"

Lastly, explaining differences in behaviour and what operations are unsupported is useful here too. From a glance I see these types of tests disabled:
1. Foreign key
2. Generated columns
3. Spatial
4. Geometric

Am I correct to assume RocksDB does not support the functionality for these tests @wangzihuacool? Let's make this as clear as possible to the reader 🙇

Reply:
Yes, RocksDB does not support these features, especially Gap lock. I'll add description to make it clear to the reader.